lib/execution_engine2/sdk/EE2Runjob.py

```diff
@@ -579,15 +580,25 @@ def _retry(self, job_id: str, job: Job, parent_job: Job, as_admin: bool = False)
                     exception=e,
                 )
 
-        # 2) Notify the retry_parent that it has been retried
+        # 2) Notify the retry_parent that it has been retried by adding a retry id
         try:
-            job.modify(inc__retry_count=1)
+            job.modify(push__retry_ids=retry_job_id)
```

Comment:
I missed this in my prior review - this `$push` and the batch parent modification `$push` above need to be `$addToSet` to avoid duplicate IDs in the list

Reply:
this is done

Reply:
:+1: